examples/typescript-esm/multisig_v2.ts

```diff
@@ -141,19 +142,39 @@ const createMultiSigTransferTransaction = async () => {
     aptosConfig: config,
   });
 
-  // Simulate the transfer transaction to make sure it passes
-  const transactionToSimulate = await generateRawTransaction({
-    aptosConfig: config,
-    sender: owner2.accountAddress,
-    payload: transactionPayload,
-  });
-
-  const simulateMultisigTx = await aptos.transaction.simulate.simple({
-    signerPublicKey: owner2.publicKey,
-    transaction: new SimpleTransaction(transactionToSimulate),
-  });
-
-  console.log("simulateMultisigTx", simulateMultisigTx);
+  // The simulation enhancement feature is not enabled on the devnet yet.
```

Comment:
can be removed. it is on devnet https://aptos-org.slack.com/archives/C05NLAKJM9U/p1726612173049729

Reply:
Done removing it.